tests/integration/Owncloud/OcisPhpSdk/OcisTest.php

```diff
@@ -30,6 +30,88 @@ public function testServiceUrlTrailingSlash(): void
         $this->assertTrue((is_array($drives) && count($drives) > 1));
     }
 
+    public function testGetAllDrives(): void
+    {
+        $ocis = $this->getOcis('admin', 'admin');
+        $sportDrive = $ocis->createDrive('Sport Project Drive');
+        $this->createdDrives[] = $sportDrive->getId();
+        $managementDrive = $ocis->createDrive('Management Project Drive');
+        $this->createdDrives[] = $managementDrive->getId();
+        $drives = $ocis->getAllDrives();
+        $this->assertCount(4, $drives);
+        foreach ($drives as $drive) {
+            $this->assertInstanceOf(Drive::class, $drive);
+            $this->assertIsString($drive->getId());
+            $this->assertIsString($drive->getName());
+            $this->assertIsObject($drive->getType());
+            $this->assertIsObject($drive->getRoot());
+        }
+        $this->assertTrue([$managementDrive->getName(),$sportDrive->getName()] === [$drives[1]->getName(),$drives[3]->getName()]);
+    }
+
+    public function testGetAllDrivesProjectType(): void
+    {
+        $ocis = $this->getOcis('admin', 'admin');
+        $sportDrive = $ocis->createDrive('Sport Project Drive');
+        $this->createdDrives[] = $sportDrive->getId();
+        $managementDrive = $ocis->createDrive('Management Project Drive');
+        $this->createdDrives[] = $managementDrive->getId();
+        $drives = $ocis->getAllDrives(
+            DriveOrder::NAME,
+            OrderDirection::ASC,
+            DriveType::PROJECT
+        );
+        $this->assertCount(2, $drives);
+        $this->assertTrue([$managementDrive->getName(),$sportDrive->getName()] === [$drives[0]->getName(),$drives[1]->getName()]);
+    }
+
+    public function testGetMyDrive(): void
+    {
+        $ocis = $this->getOcis('admin', 'admin');
+        $myDrive = $ocis->getMyDrives();
```

Comment:
here it would be good to have other drives and make sure only the drives where a specific user has access are listed